addons/base_import/models/base_import.py

```diff
@@ -167,6 +232,24 @@ def get_fields(self, model, depth=FIELDS_RECURSION_LIMIT):
                 Fields with no sub-fields will have an empty list of
                 sub-fields.
 
+            .. attribute:: field_model (str)
+
+                Used in the Odoo Field Tooltip on the import view
+                and to get the model of the field of the related field(s).
+                (notably to get possible values of selection fields)
+                    Example with crm.team model :
+                        name -> field_model = crm.team
+                        message_ids/moderation_status -> field_model = mail.message
```

Comment:
WTF I really don't understand.  This attribute just brings confusion between a field's model and comodel.  Why don't you use both `model_name` and `comodel_name` (like on field definitions) ?